main.go

```diff
@@ -124,6 +125,7 @@ func main() {
 	pflag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	pflag.BoolVar(&createRBACPermissions, "create-rbac-permissions", false, "Automatically create RBAC permissions needed by the processors")
```

Comment:
did we discuss whether it should be disabled by default? 

I would prefer to ship the OSS distro with as much enabled functionality.